### ISSUE TITLE
Custom maps

### ISSUE THREAD

**Issue by gaiagame** [thecodingmachine/workadventure#272]
Hello, I have been trying my hand in creating a new map and have encountered a couple of question/issues. (I am github newbie so likely did something wrong...) 
while I managed to get the Jitsi and a whiteboard integration I was not able to integrate our home page imaginaction.org 
![image](https://user-images.githubusercontent.com/71192199/93073730-a9509280-f683-11ea-8988-c186a8a628bf.png) 
and a youtube playlist I wanted to present. I am not sure why..
Also I am not sure where in the git add the other floor json layers. 
 
![image](https://user-images.githubusercontent.com/71192199/93074033-16fcbe80-f684-11ea-8608-6b42340090a6.png)
thanks for the input and this lovely piece of software!

**Comment by moufmouf** [comment]
Hey @gaiagame!

Beautiful map, glad you like it!


Regarding the other floors, you can put them anywhere you want, but then, you need to be able to reference them correctly from your map.

In your case, the file hierarchy looks like this:

- map.json
- Floor1/floor1.json

But in the "exitSceneUrl" parameter, you put: `../Floor1/floor1.json`
The ".." is the error (it will go up one level in the directory hierarchy and you don't need to.

Replace `../Floor1/floor1.json` with `Floor1/floor1.json` and this should work.

Regarding integrating your home page imaginaction.org, did you manage to have it working? If not, could you tell me where to look on the map?

**Comment by jacquesbh** [comment]
Hi!

The website imaginaction.org responds with the header `X-Frame-Options: SAMEORIGIN` which will deny any iframe integration outside the same domain.

If you can change the value of the header in the response from the server then you may have a solution on that matter!

**Comment by gaiagame** [comment]
Thank you both. I will try this out. Also in regards of being having people use Mobile phones browser (which be a great thing for inclusivity), I was thinking that maybe a simple app with keyboard and arrows could be nice. I tested connecting a Bluetooth keyboard to my phone and the mechanism worked just fine. So I believe it is a just a question of having an app which forces a simple keyboard with arrows and letters (maybe use the framework of anysoftkeyboard to add a dedicated keyboard?) Thanks again, everyone .

**Comment by moufmouf** [comment]
@gaiagame Yes, a mobile version is clearly on the todo list, but don't expect it before a couple of months, we have a huge list of features / improvements to add before :D

**Comment by gaiagame** [comment]
Wonderful news. and thanks for all the tips I really see a lot of potential of this in my field (performance  and social action). 
I did manage to enable iframe in the game but not able to enable the camera when I embed the map in our website. You know why ? (here is the link https://imaginaction.org/campus) 
Also I did remove the ... from the link and placed the other floor maps both in the main folder and in back/src/assets/maps and still I am not able to load them... what is it I am doing wrong? 

Thanks for all the help

Uri
